sentinel-no-op/publish.gradle

```diff
@@ -46,9 +46,9 @@ afterEvaluate {
                     }
                     developers {
                         developer {
-                            id = 'knobtviker'
-                            name = 'Bojan Komljenović'
-                            email = 'bojan.komljenovic@infinum.com'
+                            id = 'Infinum'
+                            name = 'Infinum Inc.'
+                            email = 'team.android@infinum.com'
```

Comment:
I think we might even remove email 
And move to 
```
organization {
                        name = 'Infinum Inc.'
                        url = 'https://infinum.com'
                    }
```

instead of developer

Reply:
I removed email from developer info. Since repos are public there is no barrier in reporting issue on lib.

Also, IMO we should only have _**organization**_ info. But I found that some tools that collect info for libs sometimes read only developers info and sometimes developer and organization infos are interchangeable.
This is why I added both developer and organization

Reply:
Makes sense, I agree 👍